Log.cpp

```diff
@@ -31,10 +31,11 @@ Log::~Log()
 }
 
 
-static const std::map<LogLevel, std::string> logLevelStrings = {{LogLevel::Error, "  [ERROR] "},
-	 {LogLevel::Warning, "[WARNING] "},
-	 {LogLevel::Info, "   [INFO] "},
-	 {LogLevel::Debug, "  [DEBUG]     "}};
+static const std::map<LogLevel, std::string> logLevelStrings = {{LogLevel::Error, "   [ERROR] "},
+	 {LogLevel::Warning, " [WARNING] "},
+	 {LogLevel::Info, "    [INFO] "},
+	 {LogLevel::Debug, "   [DEBUG]     "},
+    {LogLevel::Progress, "[PROGRESS] "}};
```

Comment:
Oddly, ended up with spaces instead of tabs here. I'm sure it looks right in Visual Studio, but it looks strange here.

Reply:
That would be clang I think. When formatting these it puts a tab and a space before the open curly. No idea why.